ale/isd_generate.py

```diff
@@ -190,6 +193,71 @@ def file_to_isd(
 
     return
 
+def write_json_file(isd_string, json_file):
+    """
+    Writes out json file with inputted isd string.
+    
+    Parameters
+    ----------
+    isd_string : str
+        The ISD as a JSON formatted string
+
+    json_file : str
+        Output json file path
+    """
+    with open(json_file, 'w') as fp:
+        json.dump(isd_string, fp, cls = AleJsonEncoder)
+
+def compress_isd(uncompressed_isd_file):
```

Comment:
maybe a semantics, but the function seems to compress any json file, not just ISDs. We could add an extra step that verifies that the file is at least json with a json.load or similar, we could probably call this `compress_file` or `compress_json` if we keep the extension check.